meteor/client/ui/Settings/ShowStyleBaseSettings.tsx

```diff
@@ -1091,6 +1223,47 @@ const HotkeyLegendSettings = withTranslation()(
 												></EditAttribute>
 											</label>
 										</div>
+										<div className="mod mvs mhs">
+											<label className="field">
+												{t('Host Key')}
+												<EditAttribute
+													modifiedClassName="bghl"
+													attribute={'hotkeyLegend.' + index + '.platformKey'}
+													obj={this.props.showStyleBase}
+													type="text"
+													collection={ShowStyleBases}
+													className="input text-input input-l"
+												></EditAttribute>
+											</label>
+										</div>
+										<div className="mod mvs mhs">
+											<label className="field">{t('Source Layer type')}</label>
+											<EditAttribute
+												modifiedClassName="bghl"
+												attribute={'hotkeyLegend.' + index + '.sourceLayerType'}
+												obj={this.props.showStyleBase}
+												type="dropdown"
+												options={SourceLayerType}
+												optionsAreNumbers
+												collection={ShowStyleBases}
+												className="input text-input input-l dropdown"
+												mutateUpdateValue={(v) => (v ? v : undefined)}
+											/>
+										</div>
+										<div className="mod mvs mhs">
+											<label className="field">
+												{t('Key color')}
+												<EditAttribute
+													modifiedClassName="bghl"
+													attribute={'hotkeyLegend.' + index + '.buttonColor'}
```

Comment:
I don't think this is used anywhere outside of the private TV2 Keyboard map feature, that we're apparently not importing. If it's not being used, it should be removed.

Reply:
Is any part of the hotkey legend something you can use at all? If not, I think maybe we should exclude the entire HotkeyLegendSettings

Reply:
We're keeping this in the spirit of keeping our codebases aligned as much as possible and even though NRK won't use most parts of it it won't hurt us to keep it.
We've put it behind a feature-flag for now, to reduce noise to the user in GUI.